src/phoenix/db/alembic.ini

```diff
@@ -104,7 +104,7 @@ handlers =
 qualname = sqlalchemy.engine
 
 [logger_alembic]
-level = WARN
+level = INFO
```

Comment:
Curious, is there a way of setting the logging level via environment variables?

Reply:
we can ignore the alembic config if we want but yeah I'd like to be able to set the level overall.